cmake/MacOSX/fix_dependencies.rb

```
@@ -13,511 +13,504 @@
+    ##  Currently we just copy the plugins and hope they have dependencies that are covered
+    ##  by the usual libs. Should work for Qt (our only plugin).
+    if !@plugin_dir.nil?
+      debug "HANDLING PLUGIN DIR"
+      for content in Dir.glob("#{@plugin_dir}/**/*.dylib")
+        debug "Handle dylib #{@plugin_dir + content}"
+        handleDyLib(@plugin_dir + content, File.dirname(@plugin_dir + content), [])
       end
-    else
-      debug "Skipped #{$bin_dir + content} -- No binary or object?"
     end
   end
```

RuboCop (1.82.1)

[convention] Assignment Branch Condition size for `run` is too high. [<7, 53, 20> 57.08/23]

(Metrics/AbcSize)